rbatis/src/main.rs

```diff
@@ -0,0 +1,47 @@
+#[macro_use]
+extern crate rbatis_macro_driver;
+#[macro_use]
+extern crate lazy_static;
+
+use actix_web::{get,App,HttpServer,HttpResponse,Responder,middleware::Logger};
```

Comment:
This example is missing spaces all over the place. Is there no CI job for `cargo fmt -- --check`?